Sources/SwiftGen/main.swift

```diff
@@ -32,6 +32,7 @@ let main = Group {
     $0.addCommand("run", "run commands listed in the configuration file", ConfigCLI.run)
     $0.addCommand("init", "create an initial configuration file", ConfigCLI.create)
     $0.addCommand("doc", "open the documentation for the configuration file on GitHub", ConfigCLI.doc)
+    $0.addCommand("generate-xcfilelists", ConfigCLI.generateXCFileListsDescription, ConfigCLI.generateXCFileLists)
```

Comment:
Put the description here directly (for consistency). You can use multiline strings to avoid line length warnings, like so:
```swift
"""
Lorem ipsum dolor sit amet, consectetur adipiscing elit, \
sed do eiusmod tempor incididunt ut labore 
"""
```